plugins/woocommerce-admin/client/activity-panel/activity-panel.js

```diff
@@ -246,7 +246,10 @@ export const ActivityPanel = ( { isEmbedded, query } ) => {
 				if ( currentLocation !== homescreenLocation ) {
 					// Ensure that if the user is trying to get to the task list they can see it even if
 					// it was dismissed.
-					if ( setupTaskListHidden === 'no' ) {
+					if (
+						setupTaskListHidden === 'no' ||
+						setupTaskListHidden === false
```

Comment:
> We have more "setup" task lists now than when we started and this logic is slightly dated.

Yeah, kinda confusing in relation to the experimental task lists, in this case that part doesn't matter as the hide/show setting is the same for all 4? of them.
In theory we might want some thing more like `getActiveSetupTaskList` instead of passing in the `setup` id every where, but that should not be done as part of this fix.